utils/testgen.py

```diff
@@ -219,6 +219,9 @@ def pytest_generate_tests(metafunc):
         parametrize tests in the 'module' scope.
 
     """
+
+    metafunc.function = pytest.mark.uses_testgen()(metafunc.function)
```

Comment:
pytest.mark doesn't wrap, it annotates. This can be `pytest.mark.uses_testgen(metafunc.function)`.